contributors.html

```diff
@@ -125,9 +125,7 @@
           </a></div>
 
         </div>
-        <a href="https://github.com/fossasia/labyrinth">
-          <h1 class="brand-logo center">Labyrinth</h1>
-        </a>
+        <h1 class="brand-logo center">Labyrinth</h1>
```

Comment:
What is the Purpose of this? I think the Previous one is Fine.

Reply:
@naveentherocker The logo of a website in generally linked to its main page but never to the project files directly therefore I thought of adding a contribute option to look more professional.